### PR TITLE
in_thermal: initial documentation for thermal input plugin

### PR DESCRIPTION
**Pull request by nigels-com**
In relation to proposed thermal input plugin:
https://github.com/fluent/fluent-bit/pull/1509

Signed-off-by: Nigel Stewart <nigels@nigels.com>

### CODE REVIEW

File: input/thermal.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+| type | The type of the thermal zone, such as _x86\_pkg\_temp_ |
+| temp | Current temperature in celcius |
+
+## Configuration Parameters
+
+The plugin supports the following configuration parameters:
+
+| Key | Description |
+| :--- | :--- |
+| Interval\_Sec  | Polling interval \(seconds\).  default: 60 |
```
Comment (nigels-com) [inline input/thermal.md]: `default: 1`

**Comment by edsiper** [comment]
thanks, would you please link the doc file in SUMMARY.md ?, otherwise it won't be rendered.

**Comment by nigels-com** [comment]
SUMMARY.md updated.

**Comment by edsiper** [comment]
Thanks

**Pull request merged**